src/model/users.rs

```diff
@@ -384,11 +426,6 @@ pub struct User {
 	/// Base info that's shared across different user responses
 	#[serde(flatten)]
 	pub base: AccountData,
-	/// If the user has avatar cloning on
-	pub allow_avatar_copying: bool,
-	#[serde(rename = "date_joined", with = "crate::date_format")]
-	/// When the user joined VRC
-	pub date_joined: time::Date,
```

Comment:
It takes quite a while to compare the implications of this moving to `AccountData`; Basically this is now available for the current user and from the friends endpoint as well? ... I'll bother to re-setup my testing creds to validate that I guess, since iirc lots of things weren't actually available from the friends endpoint...

Could also have updated the `docs/users.d2` diagram to make it a bit easier to visualize :stuck_out_tongue: but I guess I can do it after this gets merged as well...

Reply:
I didn't check anything but the current user/account data, so that might not be the correct change, but there are a lot of structs that have changed significantly in the last ~9 months. Some tests for data response serialization should probably be added to catch response changes.